app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java

```diff
@@ -7,14 +7,16 @@
 import android.view.Window;
 import android.view.WindowManager;
 
+import ingsw.group1.findmyphone.Manager;
+import ingsw.group1.findmyphone.R;
+
+/**
+ *
+ */
 public class AlarmAndLocateResponseActivity extends AppCompatActivity {
-    private final String AlarmAndLocateActivityTAG = "Alarm&LocateActivityTAG";
-    private String receivedTextMessage;
-    private String receivedMessageAddress;
-    private Constants constants;
-    private Manager manager;
-    private MediaPlayer mediaPlayer;
 
+    private Manager manager;
+    MediaPlayer mediaPlayer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)